page.codeberg.petsoi.wordle.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "page.codeberg.petsoi.wordle",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "wordle",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "modules": [
+        {
+            "name": "wordle",
+            "buildsystem": "meson",
+            "build-options": {
+                "append-path": "/usr/lib/sdk/rust-stable/bin"
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://codeberg.org/petsoi/wordle.git",
+                    "tag": "v0.1"
```

Comment:
your desktop file has what appstream considers "low quality" categories only, see https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#categories-and-keywords

Reply:
I hope it is ok now.